packages/widgets-manager/src/manager.spec.ts

```diff
@@ -86,12 +104,36 @@ const mockApp3: App = {
   name: 'Markdown BCI 2',
   slug: 'markdown-bci-2',
   payload_type: 'markdown',
-  payload: '',
+  payload: '{}',
   settings: {
     css: '',
     inlineCss: {},
     position: mockPositionRelativeToPosition2,
     size: {} as any,
+    queryParams: {} as any,
+  },
+  organization_id: 1,
+  source: '',
+  initialData: {} as any,
+}
+
+const mockPopupApp: App = {
+  id: 4,
+  name: 'Webrtc screenshare',
+  slug: 'bci-screenshare',
+  payload_type: 'lt-webrtc',
+  payload: '{}',
+  settings: {
```

Comment:
Please add an issue for:

- Doesn't mount the app if a param is missed
- Report in console.log which param is missed

Reply:
https://github.com/lets-talk/chat-utils/issues/58